src/Microsoft.PowerShell.Commands.Utility/commands/utility/ConvertTo-Html.cs

```diff
@@ -198,6 +199,28 @@ public SwitchParameter Fragment
 
         private SwitchParameter _fragment;
 
+        /// <summary>
+        /// Gets or sets the _hyperlink switch, if true, then auto
+        /// generate hyperlinks for URI.
+        /// </summary>
+        [Experimental("Microsoft.PowerShell.Utility.Autohyperlink", ExperimentAction.Show)]
+        [Parameter(ParameterSetName = "Autohyperlink")]
+        [ValidateNotNullOrEmpty]
+        public SwitchParameter Autohyperlink
```

Comment:
Using `Format-Table -AutoSize` as an example, we should call his `AutoHyperlink`